sass/px-file-upload.scss

```diff
@@ -27,42 +25,61 @@ $inuit-enable-btn--secondary: true;
 @import "px-actionable-design/_objects.actionable.scss";
 
 // Components
-.icon {
-  width: 16px;
-  height: 16px;
+.file-upload-fileTable {
+  display: table;
+  margin-top: $inuit-base-spacing-unit--small;
+  margin-bottom: $inuit-base-spacing-unit--large;
 }
-.fileTable {
-  margin-top: 10px;
-  margin-bottom: 20px;
+.file-upload-tr {
+  display: table-row;
 }
-.fileTable td {
-  height: 40px;
+.file-upload-tr > .file-upload-td:first-child, .file-upload-tr > .file-upload-th:first-child{
+  padding-left: $inuit-base-spacing-unit;
 }
-.fileTable, .fileTable th, .fileTable td {
+.file-upload-tr > .file-upload-td:last-child{
+  padding-right: $inuit-base-spacing-unit;
+}
+.file-upload-tr:last-of-type .file-upload-td  {
+  border-bottom: 1px solid var(--px-table-border-color, black);
+}
+.file-upload-th, .file-upload-td {
+  display: table-cell;
   text-align: left;
-  padding-left: 30px;
+  vertical-align: middle;
+  padding: $inuit-base-spacing-unit--tiny;
+  padding-left: $inuit-base-spacing-unit--huge;
+}
+.file-upload-th {
+  border-bottom: 1px solid var(--px-table-border-color, black);
 }
-.fileTable tr > td:first-child, .fileTable tr > th:first-child{
-  padding-left: 15px;
+.file-upload-td {
+  height: calculateRem(40px);
+  border-bottom: 1px solid var(--px-table-separator-color, lightgray);
 }
-.fileTable tr > td:last-child{
-  padding-right: 15px;
+.file-upload-icon {
+  width: calculateRem(16px);
+  height: calculateRem(16px);
 }
-.fileList {
+.file-upload-fileList {
   color: var(--px-file-upload-file-list-color, gray);
 }
-.dropZone {
-  padding: 5px;
+.file-upload-dropZone {
+  padding: $inuit-base-spacing-unit--tiny;
 }
-.dropZone.hover {
+.file-upload-dropZone.hover {
   background-color: var(--px-file-upload-dropzone-background-color, lightgray);
 }
-.validation {
+.file-upload-validation {
   color: var(--px-file-upload-validation-color, red);
 }
 
 // Trumps
+$inuit-enable-paddings--tiny : true;
 $inuit-enable-paddings--small : true;
+$inuit-enable-paddings : true;
 $inuit-enable-paddings--large : true;
+$inuit-enable-paddings--huge : true;
 $inuit-enable-margins--tiny : true;
+$inuit-enable-margins--small : true;
+$inuit-enable-margins--large : true;
```

Comment:
all of these are being used -- some in the sass and some in the html (`u-p-`, `u-pb+`, etc)